data/templates/nginx/plain/yunohost_panel.conf.inc

```diff
@@ -1,8 +1,8 @@
-# Insert YunoHost panel
-sub_filter </head> '<script type="text/javascript" src="/ynhpanel.js"></script></head>';
+# Insert YunoHost button + portal overlay
+sub_filter </head> '<script type="text/javascript" src="/ynh_portal.js"></script><link type="text/css" rel="stylesheet" href="/ynh_overlay.css"></link><script type="text/javascript" src="/ynhtheme/custom_portal.js"></script><link type="text/css" rel="stylesheet" href="/ynhtheme/custom_overlay.css"></link></head>';
 sub_filter_once on;
 # Apply to other mime types than text/html
 sub_filter_types application/xhtml+xml;
 # Prevent YunoHost panel files from being blocked by specific app rules
-location ~ ynhpanel\.(js|json|css) {
```

Comment:
ynhpanel.json disappear ?

Reply:
Yes, I don't know what that stuff was supposed to be but it never was actually used ...

Reply:
Hum, I remember it to be necessary and to fix apps at [some point in time](https://github.com/YunoHost/yunohost/commit/bb11168c31064d6be6cba3b8f2c48bf3a70f02e8)... :sweat_smile:

Reply:
@JimboJoe : for the JS and the CSS file, yes, here I just renamed then ... but what about `ynhpanel.json` ? It's never effectively requested anywhere ...

Reply:
It was used before here https://github.com/YunoHost/SSOwat/blob/4b85679471db9dc55e005a0a962c61da749e4c32/portal/assets/js/ynhpanel.js#L256

But now it's not used anymore.

Reply:
Note: this feature could be used to display an other page with app info. I aws thinking use that for menu_ynh, to create a default menu with the app of the user...

Reply:
Alright, but I would like to avoid maintaining features that are not actually working / used ... This is not just about keeping `ynhpanel.json` in this file, since other piece of code referring to it were also deleted in SSOwat's PR for themes ... also for semantic it should be renamed to something like `userinfo.json` to reflect more its content :/.

Reply:
https://github.com/YunoHost/SSOwat/pull/133